deploy/kubernetes/overlays/examples/gpu-privilages.yaml

```diff
@@ -0,0 +1,27 @@
+# This patch adds the required hostpath mounts and privilages to initialize NVML, as well as to access devices.
```

Comment:
Maybe link to https://github.com/google/cadvisor/blob/master/docs/running.md#hardware-accelerator-monitoring from here?

Reply:
done